jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/Utils.kt

```diff
@@ -285,3 +294,59 @@ fun isCodeModernizerAvailable(project: Project): Boolean {
 }
 
 fun isGradleProject(project: Project) = !GradleSettings.getInstance(project).linkedProjectsSettings.isEmpty()
+
```

Comment:
This is fine for now. Maybe something like `TransformLogWrapper` would be more explicit. But I think its clear why these functions are inside if they need access to logger.